packages/conf/src/index.js

```diff
@@ -34,8 +34,8 @@ export const defaultConfig = {
   format: "lingui",
   rootDir: ".",
   extractBabelOptions: {
-    plugins: [],
-    presets: []
+    plugins: ["plugin"],
```

Comment:
Does it work actually? I thought it's a default config. Does babel actually receive a config to use plugin called `plugin` and preset called `preset`?

Reply:
It's `exampleConfig`, not default one :) Yes, I've verified it has accepted different value than `plugin`. I guess it just needs to find out what type fits in there. It's an interesting idea, but it won't work well in a moment someone would want to pass some other properties that are part of Babel configuration. It's definitely nonsense to write everything here.

Reply:
omg, I see it: `exampleConfig: defaultConfig,`. Who wrote this?! 😆 

Cool, make sense!